src/meshapi/admin/device.py

```diff
@@ -6,15 +8,21 @@
 
 from meshapi.admin.inlines import DeviceLinkInline
 from meshapi.models import Device
-from meshapi.widgets import UISPHyperlinkWidget
+from meshapi.widgets import ExternalHyperlinkWidget
+
+UISP_URL = os.environ.get("UISP_URL", "https://uisp.mesh.nycmesh.net")
```

Comment:
The current value in `.env.sample` (and expected by the import script) is

```
UISP_URL=https://uisp.mesh.nycmesh.net/nms
```

so you'll have `/nms` in the url twice with this setup

Reply:
I think we should change it tbh.